### DISCUSSION TITLE
StepWisePlanner is not compatible with the latest version of Microsoft.SemanticKernel

### DISCUSSION THREAD

**Discussion by srinu0681** [microsoft/semantic-kernel#4431] [Q&A]
Looks like the latest version of Microsoft.SemanticKernel.Planners.Core (1.0.0-beta8) is not compatible with the latest version of Microsoft.SemanticKernel(1.0.1). The StepWisePlanner`s constructor expects IKernel but there is no IKernel defined. Looks like it was removed. The new constructor expects Kernel type. When the new Microsoft.SemanticKernel.Planners.Core package will be released?

**Comment by evchaki** [comment]
@srinu0681 here is a blog post that can be helpful for you - https://devblogs.microsoft.com/semantic-kernel/migrating-from-the-sequential-and-stepwise-planners-to-the-new-handlebars-and-stepwise-planner/

**Reply by srinu0681** [reply]
Yes, I replaced the StepWisePlanner with FunctionCallingStepwisePlanner and it worked fine. Thank you.
Is there any way that we can get the steps taken using FunctionCallingStepwisePlanner?

**Reply by evchaki** [reply]
@srinu0681 - check out this sample for getting telemetry - https://github.com/microsoft/semantic-kernel/tree/main/dotnet/samples/TelemetryExample
